### PR TITLE
Engin  

### PR DESCRIPTION
**Pull request by bengin34**
I've moved the whatWeDoAndWhy.js file to the components folder, as we discussed earlier.

I also added the ourMission page, but it has a few issues:

I can't adjust the image sizes to match those in the PDF.
Some SVG icons have a black background.
The sizes of certain SVG icons aren't consistent with the others.

### CODE REVIEW

**Comment by lucaCambi77** [comment]
@busratok can you please help us here? I've seen you have been doing similar work for the Header component thanks!

File: src/components/ourMission.tsx [hunk 1]
```diff
@@ -1,20 +1,105 @@
...
-        <div className="flex-1/2">a</div>
-      </div>
-
-      {/*Right of Education */}
-      <div className="flex-1/2">a</div>
-    </div>
-  );
+    return (
+        <div
+            className="flex flex-col w-full h-screen bg-no-repeat bg-cover "
```
Comment (busratok) [inline src/components/ourMission.tsx:51]:
```suggestion
            className="relative flex flex-col w-full h-screen bg-no-repeat bg-cover "
```

File: src/components/ourMission.tsx [hunk 2]
```diff
@@ -1,20 +1,105 @@
...
+            style={{
+                backgroundImage: `url(${backgroundImage.src})`,
+                backgroundPosition: "center center",
+                    // opacity:"0.7"
+            }}
+        >
```
Comment (busratok) [inline src/components/ourMission.tsx:57]:
```suggestion
        >
            <div
        className="absolute inset-0"
        style={{ backgroundColor: "rgba(47, 61, 86, 0.75)" }}
      ></div>
```

File: src/components/ourMission.tsx [hunk 3]
```diff
@@ -1,20 +1,105 @@
...
+            <div className="h-1/4 flex flex-col justify-evenly text-white items-center">
```
Comment (busratok) [inline src/components/ourMission.tsx:58]:
```suggestion
            <div className="relative z-10 h-1/4 flex flex-col justify-evenly text-white items-center">
```

File: src/components/ourMission.tsx [hunk 4]
```diff
@@ -1,20 +1,105 @@
...
+                <div className="w-[50%] flex text-center justify-center items-center">
+                    People with passion can change the world for the better. The people
+                    who are crazy enough to think they can change the world are the ones
+                    who do.
+                </div>
+                <div className=" w-full text-center ">- Steve Jobs</div>
+                <div className="w-36 border border-b-8 border-white rounded m-auto my-4 mb-1"></div>
+            </div>
+            {/* icons */}
+            <div className="h-1/4 flex flex-row text-white  w-[80%] mx-auto ">
```
Comment (busratok) [inline src/components/ourMission.tsx:72]:
```suggestion
            <div className="relative z-10 h-1/4 flex flex-row text-white  w-[80%] mx-auto ">
```

File: src/components/ourMission.tsx [hunk 5]
```diff
@@ -1,20 +1,105 @@
...
+                        key={index}
+                        imagePath={item.imagePath}
+                        content={item.content}
+                        iconPath={item.iconPath}
+                    />
+                ))}
+            </div>
+
+            {/*Right of Education */}
+            <div className="h-1/2 w-[50%] flex flex-col mt-24 justify-evenly mx-auto items-center text-white">
```
Comment (busratok) [inline src/components/ourMission.tsx:84]:
```suggestion
            <div className="relative z-10 h-1/2 w-[50%] flex flex-col mt-24 justify-evenly mx-auto items-center text-white">
```

**Review by busratok — CHANGES_REQUESTED** [review]

**Pull request closed without merging**